README.md

```diff
@@ -339,6 +339,13 @@ const app = express();
 const httpServer = ViteExpress.listen(app, 3000, () => console.log("Server is listening!"));
 ```
 
+Emits an event (`vite:close`) when the underlying Vite dev server is finished closing, since that happens asynchronously with closing the Express HTTP server.
```

Comment:
How about something like:

When HTTP server has been closed using [`close()`](https://www.w3schools.com/nodejs/met_server_close.asp) method, **vite-express** also closes **Vite** dev server asynchronously. After that, `vite:close` event is emitted so you can listen to that event to have a guarantee that all server resources were freed.

Reply:
Sounds good to me

Reply:
Perfect!